linkerd/assets/dashboards/overview.json

```diff
@@ -18,7 +18,7 @@
     "description": "## Linkerd\n\nThis dashboard provides a high-level overview of your Linkerd cluster so you can monitor its performance and route, control, and connection metrics.\n\nFor further reading on monitoring Envoy, see:\n\n- [Official Linkerd integration docs](https://docs.datadoghq.com/integrations/linkerd/)\n\nClone this template to make changes and add your own graphs and widgets.",
     "disableCog": false,
     "disableEditing": false,
-    "icon": "/static/images/saas_logos/small/linkerd@2x.png",
+    "icon": "",
```

Comment:
```suggestion
    "icon": "/static/images/saas_logos/small/linkerd.png",
```

This should work

Reply:
https://github.com/DataDog/integrations-core/pull/5402#discussion_r364938711

Reply:
Maybe we can remove `icon` field instead of `"icon": ""`. WDYT?